pack/fhv/fhvc.json

```diff
@@ -165,10 +446,266 @@
     "quantity": 1,
     "slot": "Ally",
     "subname": "Faithful Steed",
-    "text": "Your first action each round does not provoke attacks of opportunity.\n[action] Exhaust Ajax: Move. Either move twice, or choose any [[Field]] location and move to it.\n[action] If Ajax is at your location: Take control of him. Any player at Ajax's location may trigger this ability.",
+    "text": "Your first action each round does not provoke attacks of opportunity.\n[action] Exhaust Ajax: <b>Move.</b> Either move twice, or choose any [[Field]] location and move to it.\n[action] If Ajax is at your location: Take control of him. Any player at Ajax's location may trigger this ability.",
     "traits": "Ally. Creature.",
     "type_code": "asset"
   },
+  {
+    "code": "10643",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 18,
+    "enemy_damage": 3,
+    "enemy_evade": 3,
+    "enemy_fight": 5,
+    "enemy_horror": 2,
+    "faction_code": "mythos",
+    "health": 6,
+    "illustrator": "Peter Polach",
+    "is_unique": true,
+    "name": "Ursine Hybrid",
+    "pack_code": "fhvc",
+    "position": 143,
+    "quantity": 1,
+    "subname": "Starving Abomination",
+    "text": "Hunter. Massive. Retaliate.\n<b>Prey</b> - Lead investigator.\nUrsine Hybrid gets +3 [per_investigator] health and ignores barriers and decoys.\nWhile this enemy is moving, each [[Field]] location is considered connected to one another.",
+    "traits": "Creature. Monster. Mutated. Elite.",
+    "type_code": "enemy",
+    "victory": 2
+  },
+  {
```

Comment:
Some of the enemies in this set have variants with different spawn instructions.

I used ids `a`, `b`, `c` to disambiguate the different versions.

Reply:
I think this is a good idea. I've commented below that maybe the position for B and C needs to be changed - something to check in terms of how the cards display on the card list once on the DB.